packages/nvidia_nat_harbor/tests/test_atif_bridge_runner.py

```
@@ -0,0 +1,288 @@
+    artifact_path = tmp_path / "trajectory.json"
+    _write_minimal_atif(artifact_path)
+
+    async def _builtin_stub(**kwargs):
+        del kwargs
+        return 0.8, {"lane": "trajectory"}
+
+    monkeypatch.setattr("nat_harbor.verifier.evaluator_adapter._run_builtin_evaluator", _builtin_stub)
+    reward, details = evaluate_artifact_sync(
+        artifact_path=artifact_path,
+        evaluator_kind="trajectory",
+        config_file="/tmp/fake-config.yml",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake-config.yml"

(S108)